dataplane/worker_gc.cpp

```diff
@@ -931,6 +938,40 @@ inline bool worker_gc_t::is_timeout(const uint16_t timestamp,
 	return ((uint16_t)(current_time - timestamp) > timeout);
 }
 
+inline uint32_t worker_gc_t::get_acl_tcp_timeout(const uint8_t flags, const dataplane::globalBase::fw_state_config_t& fw_state_config)
+{
+	if ((flags & RTE_TCP_ACK_FLAG) && (flags & RTE_TCP_SYN_FLAG))
```

Comment:
I'm not sure this compiles the right way without unnecessary branches.
Consider more straightforward:

```c++
if ((flags & (RTE_TCP_SYN_FLAG | RTE_TCP_ACK_FLAG)) == (RTE_TCP_SYN_FLAG | RTE_TCP_ACK_FLAG)) { ...
```

Or, as you check for SYN_FLAG the next branch bellow, you can unite them both:

```c++
if ((flags & RTE_TCP_SYN_FLAG) == RTE_TCP_SYN_FLAG) {
    if ((flags & RTE_TCP_ACK_FLAG) == RTE_TCP_ACK_FLAG) {
        return fw_state_config.tcp_ack_timeout;
    } else {
        return fw_state_config.tcp_syn_timeout;
    }
}
```